### PR TITLE
Agents Kit 24.05

### PR DESCRIPTION
**Pull request by drcgjung**
## Description

Upgrades the Agent Kit to 24.05 compatibility.
Introduces Ontology Modelling Development section.
Discusses Data Sovereignity Issues.
Discusses Versioning Issues.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by HFocken** [comment]
Please update changelog!

**Comment by HFocken** [comment]
Please only include current version of standards and not deprecated versions - we currently don't support backward compatibility at Catena-X (Adoption View).

File: docs-kits/kits/knowledge-agents/development-view/modelling.md [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+* The [Layers & Modules](modules) Architecture
+* Our [Reference Implementation](reference)
+* The [Deployment](../operation-view/deployment) guide
+
+## Introduction & Standards
+
+This document gives fundamental guidelines to create ontology models for being used in Catena-X (see Catena-X standard CX-0067). It provides the semantic basis for the so called Knowledge Agent Approach that utilizes semantic web technologies for federated queries in data spaces (see Catena-X standard CX-0084).
+
+* [CX-0084 Federated Queries in Dataspaces (V1.2.0 Upcoming)](https://catena-x.net/de/standard-library)
+* [CX-0084 Federated Queries in Dataspaces (V1.1.0 Superseded)](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_Januar_2024/CX-0084-FederatedQueriesInDataSpaces-v1.1.0.pdf)
```
Comment (HFocken) [inline docs-kits/kits/knowledge-agents/development-view/modelling.md:47]: Please only include only current version of standards, not deprecated ones - currently, CX does not support backward compatibility.

File: docs-kits/kits/knowledge-agents/adoption-view/intro.md [hunk 2]
```diff
@@ -30,48 +29,53 @@ title: Adoption View
...
-##### Technology Bridges
+* [Agent Plane(s) for EDC (KA-EDC)](https://github.com/eclipse-tractusx/knowledge-agents-edc) see also [KA-EDC Operation View](../operation-view/agent_edc)
+* [Binding Agent(s) for Providers (KA-RI)](https://github.com/eclipse-tractusx/knowledge-agents) see also [KA-RI Operation View](../operation-view/provider)
+* [Agent Bridges for AAS (KA-AAS)](https://github.com/eclipse-tractusx/knowledge-agents-aas-bridge) see also [KA-AAS Operation View](../operation-view/bridge)
 
-We define bridges to other digital twin approaches, such as AAS (Asset Administration Shell), such that data and service provisioning into multiple use cases will be as effortless as possible.
+### Support
 
-### Business Value
+We support use case consumers, app developers, data providers, service providers and IT/domain consultants in order to operate as economically and well-informed as possible by giving them first-class tools, documentation and feedback.
```
Comment (HFocken) [inline docs-kits/kits/knowledge-agents/adoption-view/intro.md:76]: Please refer users to the respective retro to provide actual help - the current sentence does not offer concrete guidance.

**Comment by drcgjung** [comment]
> Please update changelog!

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/872/files#diff-304158a3c7243c73c7f3bc0c40fc06de52fc64a1aba480ed9914bbe70c32732b

Not good?

**Review by HFocken — CHANGES_REQUESTED** [review]
Thank you for updating this KIT! - general comments: 

1) please update the perspective from "we" to "the Agents KIT" to stay consistent with other KITs.

2) please provide guidance / orientation for users with regards to how agents relate to digital twins / the industry core.

File: docs-kits/kits/knowledge-agents/development-view/modelling.md (hunk 1 above)
Comment (drcgjung) [reply docs-kits/kits/knowledge-agents/development-view/modelling.md]: done (although the links to the upcoming standards I had to guess)

**Comment by TomBuchertTSystems** [comment]
> Thank you for updating this KIT! - general comments:
> 
> 1. please update the perspective from "we" to "the Agents KIT" to stay consistent with other KITs.
> 2. please provide guidance / orientation for users with regards to how agents relate to digital twins / the industry core.

-->Comment Nr 1: Implemented
-->Comment Nr 2: We mention the relation to twin approach in several parts of the KIT. What do you require exactly?

-https://catenax-ng.github.io/product-knowledge/docs/adoption-view/faq -->(see point "no redundancy")

-https://catenax-ng.github.io/product-knowledge/docs/development-view/aas/bridge -->whole page dedicated to interoperability between twins and agents

-New section "modelling" (development view) -->see point "Digital Twin and Ontologies"

**Review by HFocken — APPROVED** [review]
Thank you for the clarification & changes made - good to go!

**Comment by afederl** [comment]
It seems that the last sentence in the page Operations View/Provisioning is not complete: `Note that there are two mechanisms inside the Graph Asset Description with which a skill`

**Comment by drcgjung** [comment]
> It seems that the last sentence in the page Operations View/Provisioning is not complete: `Note that there are two mechanisms inside the Graph Asset Description with which a skill`

yes, thanks for the heads up. Looks I was fallen asleep in the middle of editing. There was a whole section missing.

**Comment by afederl** [comment]
Some more:

1. In the example request on page Development View/API/Register Skill is italian language used. Is this supposed to?
2. In the currently added section in page Operations View/Provisioning, the property `sh:shapesGraph` within last code-block (Health Indicator function graph asset) contains the whole function shape (PrognosisFunctionShape) two times.
3. In the page Development View, the paragraph "Binding Agent" contains the whole paragraph "Graph Assets" (text word by word repeated). This may be unnecessary.
4. I don't remember where exactly (I let you know if I see it again), but there where two more thins: an extra word in a sentence that don't make sense to me and a half of a sentence that is repeated at the end of a paragraph.

**Comment by drcgjung** [comment]
> Some more:
> 
> 1. In the example request on page Development View/API/Register Skill is italian language used. Is this supposed to?

chose a better example.

> 2. In the currently added section in page Operations View/Provisioning, the property `sh:shapesGraph` within last code-block (Health Indicator function graph asset) contains the whole function shape (PrognosisFunctionShape) two times.

well noticed. was also in the modelling.md (and in the E2E test ;-) - doesnt harm at runtime, but may confuse readers.

> 3. In the page Development View, the paragraph "Binding Agent" contains the whole paragraph "Graph Assets" (text word by word repeated). This may be unnecessary.

yup, removed.

> 4. I don't remember where exactly (I let you know if I see it again), but there where two more thins: an extra word in a sentence that don't make sense to me and a half of a sentence that is repeated at the end of a paragraph.

I'm sure there will be more, I will give it to mistral for checking in my holidays.

File: docs-kits/kits/knowledge-agents/development-view/modelling.md [hunk 3]
```diff
@@ -0,0 +1,714 @@
...
+* Our [Reference Implementation](reference)
+* The [Deployment](../operation-view/deployment) guide
+
+## Introduction & Standards
+
+This document gives fundamental guidelines to create ontology models for being used in Catena-X (see Catena-X standard CX-0067). It provides the semantic basis for the so called Knowledge Agent Approach that utilizes semantic web technologies for federated queries in data spaces (see Catena-X standard CX-0084).
+
+* [CX-0084 Federated Queries in Dataspaces (V1.2.0)](https://catena-x.net/de/standard-library/Update_Mai_2024/CX-0084-FederatedQueriesInDataSpaces-v1.2.0.pdf)
+* [CX-0067 Ontology Models to Realize Federated Queries in Catena-X (V1.1.0)](https://catena-x.net/de/standard-library/Update_Mai_2024/CX-0067-OntologyModelsinCatenaX-v1.1.0.pdf)
+
```
Comment (jSchuetz88) [inline docs-kits/kits/knowledge-agents/development-view/modelling.md:48]: As the URLs to the PDF-files won't work anymore after the Release 24.05, I recommend to update the URL to: https://catena-x.net/de/standard-library

**Review by jSchuetz88 — APPROVED** [review]
Thank you for the contribution, only minor remarks / questions from my side. Good to go.

**Comment by danielmiehle** [comment]
LGTM! @drcgjung can you please check the 3rd party dependencies issues? We would like to merge the PR today. Thanks!

**Review by danielmiehle — APPROVED** [review]

**Review by maximilianong — APPROVED** [review]

**Review by obalandi — COMMENTED** [review]
LGTM

**Pull request merged**